datafusion/core/src/optimizer/projection_push_down.rs

```diff
@@ -432,6 +432,34 @@ fn optimize_plan(
                 alias: alias.clone(),
             }))
         }
+        LogicalPlan::SubqueryAlias(SubqueryAlias { input, alias, .. }) => {
```

Comment:
is it worth a test for projection pushdown specifically?

Reply:
Good point. Test added.